app/views/publish/courses/skilled_worker_visa_sponsorship/_form_fields.html.erb

```diff
@@ -6,11 +6,16 @@
       </h1>
     </legend>
```

Comment:
I think the markup for this form is a bit dodgy. I thought I updated it but it looks i only fixed it for the student visas (if you rebase). For this one, i don't think the heading should be wrapped in a legend. Eg https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/views/publish/courses/student_visa_sponsorship/_form_fields.html.erb#L2

Reply:
https://design-system.service.gov.uk/components/radios/default/index.html
You sure about h1 nested in fieldset?

Reply:
moving h1 out